docs/docs/cmd/onenote/notebook/notebook-list.md

```diff
@@ -68,3 +68,59 @@ m365 onenote notebook list --webUrl https://contoso.sharepoint.com/sites/testsit
 ## More information
 
 - List notebooks (MS Graph docs): [https://docs.microsoft.com/en-us/graph/api/onenote-list-notebooks?view=graph-rest-1.0&tabs=http](https://docs.microsoft.com/en-us/graph/api/onenote-list-notebooks?view=graph-rest-1.0&tabs=http)
+
+## Response
+
+=== "JSON"
+
+    ```json
+    [
+      {
+        "id": "1-db46efb2-1b80-4a28-987d-507cff465418",
+        "self": "https://graph.microsoft.com/v1.0/users/cfa02cf8-e903-4b76-a17b-15e43d4cf81e/onenote/notebooks/1-db46efb2-1b80-4a28-987d-507cff465418",
+        "createdDateTime": "2017-11-24T11:00:46Z",
+        "displayName": "TeamsNotebook(Shared)",
+        "lastModifiedDateTime": "2017-11-24T11:00:46Z",
+        "isDefault": false,
+        "userRole": "Owner",
+        "isShared": false,
+        "sectionsUrl": "https://graph.microsoft.com/v1.0/users/cfa02cf8-e903-4b76-a17b-15e43d4cf81e/onenote/notebooks/1-db46efb2-1b80-4a28-987d-507cff465418/sections",
+        "sectionGroupsUrl": "https://graph.microsoft.com/v1.0/users/cfa02cf8-e903-4b76-a17b-15e43d4cf81e/onenote/notebooks/1-db46efb2-1b80-4a28-987d-507cff465418/sectionGroups",
+        "createdBy": {
+          "user": {
+            "id": "cfa02cf8-e903-4b76-a17b-15e43d4cf81e",
+            "displayName": "Demo User1"
+          }
+        },
+        "lastModifiedBy": {
+          "user": {
+            "id": "cfa02cf8-e903-4b76-a17b-15e43d4cf81e",
+            "displayName": "Demo User1"
+          }
+        },
+        "links": {
+          "oneNoteClientUrl": {
+            "href": "onenote:https://contoso-my.sharepoint.com/personal/user1_contoso_com/Documents/Notebooks/TeamsNotebook(Shared)"
+          },
+          "oneNoteWebUrl": {
+            "href": "https://contoso-my.sharepoint.com/personal/user1_contoso_com/Documents/Notebooks/TeamsNotebook(Shared)"
+          }
+        }
+      }
+    ]
+    ```
+
+=== "Text"
+
+    ```text
+    createdDateTime: 2017-11-24T11:00:46Z
+    displayName : TeamsNotebook(Shared)
+    id : 1-db46efb2-1b80-4a28-987d-507cff465418
```

Comment:
Seems like something went wrong  while formatting this response.
Normally the response should look like:

![image](https://user-images.githubusercontent.com/11723921/205409649-30f8169a-26d8-49f2-bc4f-df67765d604b.png)